geoportal/c2cgeoportal_geoportal/scripts/theme2fts.py

```diff
@@ -190,7 +190,7 @@ def _add_fts(
             self.imported.add(key)
             for lang in self.languages:
                 fts = FullTextSearch()
-                fts.label = self._[lang].gettext(item.name)
+                fts.label = self._render_label(item, lang) or self._[lang].gettext(item.name)
```

Comment:
I think that's more coherent
```suggestion
                fts.label = self._render_label(item, lang)
```